buildscripts/kokoro/unix.sh

```diff
@@ -36,7 +36,6 @@ GRADLE_FLAGS+=" -Pcheckstyle.ignoreFailures=false"
 GRADLE_FLAGS+=" -PfailOnWarnings=true"
 GRADLE_FLAGS+=" -PerrorProne=true"
 GRADLE_FLAGS+=" -PskipAndroid=true"
-GRADLE_FLAGS+=" -Dorg.gradle.parallel=true"
```

Comment:
Did you check whether this had an impact? Looking at the gradle documentation it claims this is defaulting to false. But that is a bit surprising to me. https://docs.gradle.org/current/userguide/build_environment.html

Reply:
What's the expected impact? Build run time? If yes, I see no change in Linux artifacts - it's 26±3 min.

Reply:
Tried parallel build locally:
- BUILD SUCCESSFUL in 9m 49s 
- BUILD SUCCESSFUL in 6m 50s `--parallel`

I'll undo this change.